docs/reference/transform/limitations.asciidoc

```diff
@@ -277,8 +288,10 @@ documents if your {transform} contains a `group_by` based on `date_histogram`.
 
 {kibana-ref}/xpack-spaces.html[Spaces] enable you to organize your source and 
 destination indices and other saved objects in {kib} and to see only the objects 
-that belong to your space. However, this limited scope does not apply to 
-{transforms}; they are visible in all spaces.
+that belong to your space. However, a {transform} is a long running task which 
+is managed on cluster level and therefore not limited in scope to certain 
+spaces. Space awareness can be implemented by the creation of the {data-source} 
```

Comment:
> Space awareness can be implemented by the creation of the {data-source} 

Can we provide a link or more info? I didn't think space-level privileges were associated with indices or data views. However, if we're just referring to index-level privileges, we can link to https://www.elastic.co/guide/en/kibana/master/kibana-role-management.html

Reply:
@lcawl It is possible to assign Spaces to data views or indices as the screenshot below shows.
<img width="1268" alt="Screenshot 2022-05-04 at 11 12 24" src="https://user-images.githubusercontent.com/22324794/166654095-c0d17c2e-8c27-4205-b225-4ba9b5ad2ac9.png">
I provided some more info via 
[057207f](https://github.com/elastic/elasticsearch/pull/86397/commits/057207f625d451fa3a1066f395acf883b211cf3d).
Please let me know what you think!

Reply:
TIL!  Thanks for the clarification!